src/Network-MIME/MimeConverter.class.st

```diff
@@ -40,8 +40,8 @@ MimeConverter class >> mimeDecode: aStringOrStream to: outStream [
 
 { #category : #convenience }
 MimeConverter class >> mimeEncode: aCollectionOrStream [
-	^ String streamContents: [:out |
-		self mimeEncode: aCollectionOrStream to: out]
+	^ (String streamContents: [:out |
+		self mimeEncode: aCollectionOrStream to: out]) readStream
```

Comment:
it should be rather something like:
```
| writeStream |

writeStream := WriteStream on: String new.
self mimeEncode: aCollectionOrStream to: writeStream.
^ writeStream
```

It is definitely not efficient to create stream on string and then create a stream on the string content